documentation/content/en/books/handbook/virtualization/_index.adoc

```diff
@@ -272,6 +272,15 @@ To run FreeBSD smoothly on VMWare, drivers should be installed:
 ====
 The xf86 packages are only available for x86 FreeBSD guests.
 ====
+[.procedure]
+. Enabling the mouse
++
+Some users have reported issues using the mouse in the virtual machine. The mouse may be enabled by appending the following to [.filename]#/boot/loader.conf#:
```

Comment:
One sentence per line. Also, is there a reason to use loader.conf instead of the kld_list rc.conf(5) variable?

Reply:
Thanks, I put the sentence on a new line.

> Also, is there a reason to use loader.conf instead of the kld_list rc.conf(5) variable?

Yes -- this module should be loaded at boot time. From the manpages:

> To  compile this driver into the kernel, place the following lines in your kernel configuration file:
>
>	     device ums
>	     device hid
>	     device uhci
>	     device ohci
>	     device usb
>
>       Alternatively, to load the driver as a module at	boot time,  place  the following line in loader.conf(5):
>
>	     ums_load="YES"

https://man.freebsd.org/cgi/man.cgi?query=ums&sektion=4&format=html